go/src/socialapi/workers/payment/customer.go

```diff
@@ -58,7 +62,9 @@ func DeleteCustomerForGroup(groupName string) error {
 	return modelhelper.UpdateGroupPartial(
 		modelhelper.Selector{"_id": group.Id},
 		modelhelper.Selector{
-			"$unset": modelhelper.Selector{"payment.customer.id": ""},
+			// deleting customer deletes everything belong to that customer in stripe,
+			// so say we all
+			"$unset": modelhelper.Selector{"payment": ""},
```

Comment:
we had two motivations back in time 
1) we wanted one less import line and added Selector 
2) bongo in nodejs side has "selector" and "options" https://github.com/koding/koding/blob/5956d6ca5f804e58452fe2aea548328fdb8acff8/node_modules_koding/jraphical/lib/module.coffee#L19